.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "10"
+
+# Run the the validate script
+install:
+  - yarn global add bolt
```

Comment:
I think there is a thing for travis to make sure it uses yarn cache. 
https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn#caching-with-yarn